### ISSUE TITLE
After Homebridge restart, asking Siri to turn on a dimmer sets it to F0

### ISSUE THREAD

**Issue by Cardo1** [rooi/homebridge-lightwaverf#43]
Bit of an odd one I’ve been scratching my head over for a while, but I think I’ve worked out what’s going wrong.

After restarting HB, if I try to turn on any dimmers by asking Siri to turn them “on”, they won’t come on.  What I’ve discovered is HB is turning them on but at 0 brightness by sending an F0 command, such as:

Sending message: 001, !R10D1F0 |

Asking Siri to set the light to 100 works fine.

Once the light has been set either by asking for a specific brightness level, or by using an app, which will specify a brightness level, then things carry on working correctly, as asking Siri to turn the light “on” will turn it back on at the previously set brightness.

I don’t know if there’s a way for the plugin to remember the last brightness setting used by each dimmer, so after an HB restart, asking Siri to turn the light “on” will set it to whatever it was last set to, rather than F0, which won’t do anything.

Thanks!

**Comment by rooi** [comment]
Hi Cardo1

I've seem to have missed you issue. Do you still experience problems?

Thanks

**Comment by Cardo1** [comment]
Yes, this issue still manifests. Upon HB restart, asking for dimmers to turn “on” doesn’t work as they are turned on at 0 brightness.

**Comment by andygroves** [comment]
Hi all if I reply to this do I get seen

Andy Groves

Sent from my iPhone

On 23 Apr 2021, at 20:57, Cardo1 ***@***.***> wrote:

﻿

Yes, this issue still manifests. Upon HB restart, asking for dimmers to turn “on” doesn’t work as they are turned on at 0 brightness.

**Comment by Cardo1** [comment]
> Hi all if I reply to this do I get seen Andy Groves
> […](#)

Yes.

**Comment by rooi** [comment]
I just pushed a new release. The dim percentage after resorting homebridge should now be 50%. Could you please confirm?
I'll try to implement a clearer solution that saves the last dim percentage and uses that after restart.

**Comment by Cardo1** [comment]
> I just pushed a new release. The dim percentage after resorting homebridge should now be 50%. Could you please confirm?
> I'll try to implement a clearer solution that saves the last dim percentage and uses that after restart.

The dim percentage does now seem to default to 50% after a restart, thanks. This is an improvement over 0%!

Remembering the last % set would be the tidiest solution, imho.
